### ISSUE TITLE
Feature Request: Remove Block Ids Compile Step

### ISSUE THREAD

**Issue by anon3792737** [kevboh/longform#128]
I often have Character / Setting notes that refer to blocks within my manuscript.

Currently, removin the [Block Ids](https://help.obsidian.md/How+to/Link+to+blocks) is a manual process that I do after compiling (though I supposed this could be done with RegEx?)

It would be nice if there was a Compile Step that automatically removed Block Ids.

Thanks!

**Comment by kevboh** [comment]
Yes, this should be part of the existing link removal step. Thanks.
